rs/nns/governance/src/proposals/mod.rs

```diff
@@ -54,3 +59,82 @@ pub(crate) fn invalid_proposal_error(reason: &str) -> GovernanceError {
         format!("Proposal invalid because of {}", reason),
     )
 }
+
+/// Weighted voting power is just voting power * the proposal's (topic's) weight.
+///
+/// For example, suppose this returns ({42 => 3.14}, 99.9). This means that
+/// neuron 42 should get 3.14/99.9 of today's reward purse.
+///
+/// Non-essential fact: It is highly unusual for result.1 to be the sum of the
```

Comment:
Reading this comment, I was a bit puzzled about why "highly unusual" refers to: 

Is the comment explaining why the code is unusual?

Or is the comment describing an unusual scenario? (I now this this is the case)

I suggest inverting the statement: Describe what is expected in a typical case, then say that sometimes this would not happen.